web_client/stylesheets/panel.styl

```diff
@@ -2,15 +2,13 @@
 
   .s-panel
     background-color #ffffff
```

Comment:
If we want to preserve most of the opacity look, the background color could be changed to `rgba(255, 255, 255, 0.95)`.